test/unit/query.test.js

```diff
@@ -1,37 +0,0 @@
-'use strict';
```

Comment:
As noted in original PR this test was added to ensure connection is only released once and there is no `Resource is not a part of pool` error.

Now each retry fetch new connection so this test is not required. Also that error was already handled [here](https://github.com/sequelize/sequelize/blob/d65a95c953ccd0595d5b406cd198ac149e14d955/lib/dialects/abstract/connection-manager.js#L287)